EIPS/eip-4824.md

```diff
@@ -0,0 +1,220 @@
+---
+eip: 4824
+title: DAO API
```

Comment:
I'm still not sold on this title... I know it's been bikeshedded a lot, but:

 - Proposal, Membership, Activity, and Governance Interfaces for DAOs
 - DAO Metadata Interface

Reply:
It's been a long, long conversation 😅  the other co-authors and I went through a bunch of alternatives, had our own disagreements about we felt made the most sense, and we came out with DAO API.

Re: "DAO Metadata Interface", the point is that not all the data is "metadata". Would you call proposals data being updated and consumed on a daily basis the "metadata" of the DAO?

Thanks again for all your comments!

Reply:
While I don't have a clearly better suggestion, `DAO API` isn't descriptive enough for an EIP.

Here's a couple more random ones (none of them great):

 * Proposal, Membership, Activity, and Governance Interfaces for DAOs
 * Operational Interfaces for DAOs
 * Common Interfaces for DAOs
 * Universal Interfaces for DAOs

The first one is the most descriptive, but it's quite long.

Reply:
Yeah, I'm not sure how to respond. If it is the name that is holding up publishing of this EIP, at this point we are happy to change it to whatever the EIP editors want to call it.

Reply:
The name is definitely holding off the publishing of the EIP. I would also prefer the first name:
```suggestion
title: Proposal, Membership, Activity, and Governance Interfaces for DAOs
```

Reply:
Took a vote of the co-authors, and of these options, we preferred "Common Interfaces for DAOs". I've implemented that change.

We are wary of the longer name because it takes the form of a list, and there's a good chance that the elements of the list may need to change or be extended in the future, which daoURI is intended to support.